packages/app-project/src/components/ProjectHeader/components/Background/Background.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types'
+import { string } from 'react'
+import { Box } from 'grommet'
+import styled from 'styled-components'
+
+const StyledBox = styled(Box)`
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+`
+
+const StyledImg = styled.img`
+  filter: blur(4px);
+  object-fit: cover;
+  opacity: 0.5;
+`
```

Comment:
This doesn't feel right on an `img` tag, I thought `object-fit: cover` would do this, but adding `width: 100%` stretched the background image out similar to PFE's current style.

Reply:
Why not use a background image, and `background-size: cover` like we do on zooniverse.org? I'm not understanding why the background has been changed to use a content image.

Reply:
Because I was writing this after a hospital visit with Jake, and all I could remember at the time was that you can't apply CSS filters directly to background images. I'll copy over the existing PFE stuff

Reply:
:+1: